api_types.ts

```diff
@@ -513,16 +513,20 @@ export interface TemporaryBlobStorage {
    *
    * The URL expires after 15 minutes by default, but you may pass a custom expiry, however
    * Coda reserves the right to ignore long expirations.
+   *
+   * You can control what the file downloads as if opened in a browser using the `downloadAs` parameter.
```

Comment:
can you update the documentation to be more specific about filename? I thought it was file type in the first place.

Reply:
ah yes i should update the changelog too

Reply:
oh, may simply make `file` to `file name`

`what the file downloads as` I was thinking maybe you want to control what a binary blob is downloaded as jpeg or png or what file type. lol

Reply:
ahh gotcha ok ill make this more clear

Reply:
@ekoleda-codaio Hmm, is `downloadAs` actually clearer than `filename`? I feel like the latter is much more intuitive. If we go with `downloadAs`, it seems like it needs more of a qualifer, because I agree with Huayang it is ambiguous what download property this refers to.

Reply:
As you mentioned in the bug comments, it doesn't actually set the filename portion of the temporary URL, so just `filename` would be misleading. I think keeping `download` somewhere in the property name is important, but I think adding `filename` makes it clearer what the value should be. `downloadFilename` seems like a good option to me. (I think "filename" is more common than "file name" these days, and it's what the Node `fs` library uses.)

Reply:
changed to `downloadFilename